components/zoho_assist/actions/create-session/create-session.ts

```diff
@@ -0,0 +1,68 @@
+import { defineAction } from "@pipedream/types";
+import app from "../../app/zoho_assist.app";
+import { CreateSessionParams } from "../../common/types";
+
+export default defineAction({
+  name: "Create Session",
+  description: "Create a remote support or screen sharing session. [See the documentation](https://www.zoho.com/assist/api/createasession.html)",
+  key: "zoho_assist-create-session",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    app,
+    customerEmail: {
+      type: "string",
+      label: "Customer Email",
+      description:
+        "Customer Email to whom the join invitation is to be sent.",
+      optional: true,
+    },
+    type: {
```

Comment:
since the prop `type` is used in more than one action, consider implement it on the app file and use `propDefinitions`

Reply:
While they are similar, the only thing they'll share is the type and label. But I'll move it anyway. Thanks